src/modules/organization/home/team/org-event-list-card/useOrgEventListCard.ts

```diff
@@ -0,0 +1,15 @@
+export default function useOrgEventListCard(
+  attendeeLimit: number,
+  currentAttendee: number,
+  eventStatus: string
+) {
+  const participantNumber = currentAttendee + "/" + attendeeLimit;
+  const isParticipantFull = currentAttendee === attendeeLimit;
+  const isEventClosed = eventStatus === "Closed";

```

Comment:
Should we use enum status for "Close"

Reply:
You may be able to generate type for this by using `yarn generate`

Reply:
poom do not want to do